### ISSUE TITLE
signInWithPopup should return an UserCredential ?

### ISSUE THREAD

**Issue by leonardolessa** [n6g7/redux-saga-firebase#241]
I'm very unexperienced with Firebase so i'm not sure if its a legit issue.
I'm trying to grab user information from the info returned from signInWithPopup and according to the firebase docs the method should return UserCredential (https://firebase.google.com/docs/reference/js/firebase.auth.Auth.html#signinwithpopup) that would solve my problem.
But according to redux-saga-firebase the object returned is AuthCredential, which is true based on my implementation.
Is this intended? The difference from the firebase docs and the lib implementation or am I missing something?

Btw thanks for sharing the project.

**Comment by ctq3103** [comment]
I got the same problem. I have to use firebase.auth() instead. Are there any other differences between these two that you know, from your experience?

**Comment by im-thom** [comment]
Yeah I feel like this should definitely be returning a UserCredential.

@ctq3103 Another difference; and the problem I am having is that you cannot get access to things like "IsNewUser" during authentication flow. Gonna have to use the regular firebase auth flow.

If I'm misinterpreting, anyone feel free to let me know 🙃
